### PR TITLE
Feature/seals support

### PR DESCRIPTION
**Pull request by iparask**
This PR introduces the full support of the Seals use case. The package will install RADICAL Ensemble Toolkit. To execute the seals use case the following command will be executed:

```bash
iceberg -r xsede.bridges -q GPU-small -c 32 -g 2 -ip <path>/SealsImages -op ./ -w 30 -p mc3bggp seals -s 299 -m UnetCntWRN_ts-vanilla.tar -mp <path>/UnetCntWRN/ -hy A -ma UnetCntWRN
```

All paths need to be set per user.

### CODE REVIEW

File: src/iceberg/executor/seals.py [hunk 1]
```diff
@@ -6,6 +6,8 @@
 
 from __future__ import print_function
 import os
+import pandas as pd
```
Comment (mturilli) [inline src/iceberg/executor/seals.py]: Do we use Pandas only for reading a csv file? If so, should we use something lighter like https://docs.python.org/3/library/csv.html? Pandas takes a fair amount of time downloading and installing.

**Review by mturilli — COMMENTED** [review]
Left a minor comment. Code looks fine otherwise.

**Comment by iparask** [comment]
[ICEBERG Workshop](https://trello.com/c/iepvJXQz/4-iceberg-workshop)

**Comment by AymenFJA** [comment]
I tried to execute the following command :
```
iceberg --resource xsede.bridges  --queue RM --cpus 28 --gpus 1 --input_path /home/seals/ --output_path /home/seals/ --walltime 60 --project mc3bggb --analysis seals
```
and i am getting the following error :

```
usage: iceberg [-h] --resource RESOURCE [--queue QUEUE] --cpus CPUS --gpus
               GPUS --input_path INPUT_PATH --output_path OUTPUT_PATH
               --walltime WALLTIME [--project PROJECT]
               {seals,penguins,4Dgeolocation,rivers,landcover} ...
iceberg: error: unrecognized arguments: --analysis
```

Am I doing something wrong?

**Comment by iparask** [comment]
Yes, there is no analysis flag anymore. Each use case is a command.

File: src/iceberg/executor/seals.py (hunk 1 above)
Comment (iparask) [reply src/iceberg/executor/seals.py]: @mturilli I made the requested change and it works for me.

**Comment by AymenFJA** [comment]
I tried the following command : 

``` 
iceberg -r xsede.bridges -q GPU-small -c 32 -g 2 -ip <path>/SealsImages -op ./ -w 30 -p mc3bggp seals -s 299 -m UnetCntWRN_ts-vanilla.tar -mp <path>/UnetCntWRN/ -hy A -ma UnetCntWRN
```
and i can see that ` seals.0000` session is generated with EnTK log files.

**Pull request merged**